src/ActivityRepository.js

```diff
@@ -1,51 +1,26 @@
 class ActivityRepository {
-  constructor(today) {
```

Comment:
I guess since `today` isn't being used immediately in the constructor we can remove this part, right?

Reply:
Good catch! I forgot to delete this.